### PR TITLE
SplitButton menu icon fix

### LINKED ISSUE #9344: PR #9295 caused regression in menu icon disabled state for Buttons
PR: https://github.com/OfficeDev/office-ui-fabric-react/pull/9295
### Environment Information 
 
- __Package version(s)__: 6.188.0
- __Browser and OS versions__:  all

#### Actual behavior:
 
For FlyoutAnchor's the menu icon doesn't look disabled when the Button control is disabled
 
#### Expected behavior:
 
The menu icon matches the rest of the button control and has disabled styles

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes/No - The PR just need to be reverted.
 
Requested priority: (Blocking, High, Normal, Low) : Blocking
 
Products/sites affected: Office Online

### PR DESCRIPTION
**Pull request by GitHubMaxwell**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9344
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Fixed an issue in the SplitButton component where the menu icon took on a disabled color when only the primary action was meant to. The SplitButton divider colors were also revised per design comp. Lastly, the CommandBar version of the SplitButton also received these changes.

This issue was originally fixed in PR #9295 but caused issues.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9370)

### CODE REVIEW

File: common/changes/office-ui-fabric-react/v-mare-SplitButton-fix-9344_2019-06-07-05-17.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixed issue where the menu icon in the SplitButton component would take on a disabled appearance with only the primary action disabled. New screener tests were written to accommodate these changes. The CommandBarButton that is in the SplitButton configuration also took on these menu icon changes.",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/v-mare-SplitButton-fix-9344_2019-06-07-05-17.json]: nit: too much info
```suggestion
      "comment": "Fixes issue when the menu icon in the second button of the `SplitButton` gets disabled styles when only the primary action is disabled.",
```

File: apps/vr-tests/src/stories/Button.stories.tsx [hunk 2]
```diff
@@ -166,10 +166,13 @@ storiesOf('Button Split', module)
   .addStory('Root', () => <DefaultButton {...commandProps} split={true} />, { rtl: true })
   .addStory('Disabled', () => <DefaultButton {...commandProps} disabled={true} split={true} />)
   .addStory('Checked', () => <DefaultButton {...commandProps} checked={true} split={true} />)
-  .addStory('Primary', () => <DefaultButton {...commandProps} primary={true} split={true} />)
-  .addStory('Primary Disabled', () => (
+  .addStory('Primary Theme', () => <DefaultButton {...commandProps} primary={true} split={true} />)
+  .addStory('Primary Theme with Disabled State', () => (
```
Comment (Vitalius1) [inline apps/vr-tests/src/stories/Button.stories.tsx]: no changes needed for these 2 LoC (169 and 170). `Primary` in here is not the theme, it's only the type of the button that happens to use `themePrimary` palette slot for color. Revert this changes please.

Comment (Vitalius1) [reply apps/vr-tests/src/stories/Button.stories.tsx]: This way you avoid to much noise in the screener tests.

Comment (GitHubMaxwell) [reply apps/vr-tests/src/stories/Button.stories.tsx]: Thanks @Vitalius1 !

**Comment by Vitalius1** [comment]
@betrue-final-final can you accept the new screener tests please? Thanks.

**Comment by Vitalius1** [comment]
@khmakoto can we get your review and green light if everything works on your side? Thanks!

File: apps/vr-tests/src/stories/Button.stories.tsx [hunk 3]
```diff
@@ -170,6 +170,9 @@ storiesOf('Button Split', module)
   .addStory('Primary Disabled', () => (
     <DefaultButton {...commandProps} primary={true} disabled={true} split={true} />
   ))
+  .addStory('Primary with Primary Action Disabled', () => (
+    <DefaultButton {...commandProps} primaryDisabled={true} primary={true} split={true} />
```
Comment (khmakoto) [inline apps/vr-tests/src/stories/Button.stories.tsx]: Can we add a default variant with primary disabled here?

Comment (Vitalius1) [reply apps/vr-tests/src/stories/Button.stories.tsx]: @GitHubMaxwell if you add a default variant story make sure you bug Ben 😄 to get an approval as soon as the CI is done.
